tests/data/opensearch-dashboards-build-1.2.0.yml

```diff
@@ -1,6 +1,7 @@
 ---
 build:
   architecture: x64
+  distribution: None
```

Comment:
Same as for 1.1.0, remove here and everywhere, except for 1.3.0 build output that should now write a value of distribution.

Reply:
So to be clear this is to update a new test for 1.3.0 the release, not the schema?